src/components/icons/StarIcon.tsx

```diff
@@ -2,28 +2,33 @@ import { css, cx } from '../../../styled-system/css'
 import { icon } from '../../../styled-system/recipes'
 import { token } from '../../../styled-system/tokens'
 import IconType from '../../@types/Icon'
+import { ICON_SCALING_FACTOR } from '../../constants'
+
+/** Favorite icon refactored. */
+const FavoriteIcon = ({ fill, size = 14, style = {}, cssRaw }: IconType) => {
```

Comment:
The file name should always match the component name, so rename this file to `FavoritesIcon.tsx`.